docs/WritingTests.asciidoc

```diff
@@ -1605,20 +1605,39 @@ describes how downloadable assets can be specified. It is important to note
 that the specified asset is only downloaded once. New versions must be
 supplied as new, unambiguous download target file names.
 
-=== Triggering tests based on an any remote git refspec or open github pull request
+=== Triggering tests based on an any remote Git refspec or open GitHub pull request
 
 openQA also supports to trigger tests using test code from an open pull
-request on github or any branch or git refspec. That means that code changes
+request on GitHub or any branch or Git refspec. That means that code changes
 that are not yet available on a production instance of openQA can be tested
 safely to ensure the code changes work as expected before merging the code
-intro a production repository and branch. This works by setting the
+into a production repository and branch. This works by setting the
 `CASEDIR` parameter of os-autoinst to a valid Git repository path including
 an optional branch/refspec specifier. It is also possible to set `NEEDLES_DIR`
 to a valid Git repository path to use custom needles.
 See
 https://github.com/os-autoinst/os-autoinst/blob/master/doc/backend_vars.asciidoc
 for details.
 
+[NOTE]
```

Comment:
@phil-hands I would appreciate your feedback if the following documentation section would be clear enough to you to call https://github.com/os-autoinst/openQA/issues/4576 fixed

Reply:
_Regarding the PR in general:_  Great! -- I'm very much looking forward to getting it packaged, to be able to use it :-)

_Regarding the docs:_ I'm not sure that for a user who hasn't yet looked at the code, the relevance of what  `NEEDLES_DIR` ends up being set to will immediately reveal what that allows them to do,

How about adding something like:

> ... with the result that the needles that would then be used would be those to be found in `https://github.com/.../tree/master/the-needles`